src/api/app/controllers/worker/capability_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Worker::CapabilityController < ApplicationController
+  def show
+    worker = params[:worker]
+    render_error(status: 404, errorcode: 'not_found', message: 'worker not found') unless worker
```

Comment:
I think you need a `return` here

Reply:
hey @bgeuken, There is a test for that on the ```capability_controller_spec```. Reading the code from ```render_error``` looks like its not necessary... what do you think?

Reply:
I think the return is to not execute `pass_to_backend` if is not necessary.

Reply:
@bgeuken could you please review it again?